crd/crd.go

```diff
@@ -88,6 +88,10 @@ func New(config Config) (*CRD, error) {
 	return newCRD, nil
 }
 
+func (c *CRD) CreateEndpoint() string {
+	return filepath.Join("apis", c.Group(), c.Version(), c.Plural())
+}
```

Comment:
In the tests I noticed it makes sense to have a dedicated method to get the endpoint which we can use to create custom objects. Sneaking that in here.